packaging/installer/install-required-packages.sh

```diff
@@ -1550,7 +1550,7 @@ validate_tree_centos() {
     echo >&2 " > Checking for PowerTools ..."
     if ! run yum ${sudo} repolist | grep PowerTools; then
```

Comment:
no need to update grep as well or `grep -i PowerTools` ?

Reply:
Nope, the full name is in the output of `yum repolist`, which includes the capitalized version. And even if it gets a false negative it, the config-manager command below is idempotent.